README.md

```diff
@@ -1,26 +1,37 @@
 # IWC - Intergalactic Workflow Commission
 
-## This group exists to create a registry of community workflows. 
-
-### Reproducibility is important. Our goals are to:
- - foster workflow use
- - incorporate versioning
- - capture more metadata: (names, versions, authors, use cases, etc.)
- - help scientists find workflows!
-
-### Methods Proposal
- - discover workflows
- - establish quality standards (version numbers, tools exist, author is set, training available, test exists ...)
- - automated testing of workflows utilizing [Planemo](https://github.com/galaxyproject/planemo) test
- - two layers of test data:
-   - basic functionality testing (quick to run, tools exist) 
-   - downsampled data (validity of workflow results)
-
-### Import process:
- - submit a pull request (PR) that adds a link to a GA file and metadata  
- - confirm that the GA and metadata conform to standards (linting criteria to be defined)
- - test the workflow 
- - if linting passes, tests pass, and human review passes, PR is merged and added to registry list
-
-### Meeting minutes:
- * [the notes](https://github.com/galaxyproject/iwc/issues/3) from the initial meeting at GCCBOSC2018 CoFest
+[![Galaxy Workflow Linting and Tests](https://github.com/galaxyproject/iwc/actions/workflows/workflow_test.yml/badge.svg?branch=main)](https://github.com/galaxyproject/iwc/actions/workflows/workflow_test.yml/badge.svg?branch=main)
+
+## The IWC maintains high-quality Galaxy Workflows
+
+Workflows are categorized in the [workflows](workflows/) directory, and listed in [Dockstore](https://dockstore.org/organizations/iwc) and [WorkflowHub](https://workflowhub.eu/search?utf8=%E2%9C%93&q=IWC#workflows).
+
+All workflows are reviewed and tested before publication and with every new Galaxy release.
+Deposited workflows follow [best practices](https://planemo.readthedocs.io/en/latest/best_practices_workflows.html) and are versioned using github releases.
+Workflows also contain important metadata, such as:
+
+- License
+- Author
+- Institutes
+
+Additionally the IWC will collect further best practices, tips and tricks, FAQs and assist the community in designing high-quality Galaxy workflows.
+
+## Importing Workflows into Galaxy
+
+To import IWC workflows into your Galaxy instance use the TRS workflow search in the Galaxy interface.
+Click on "Workflows", "Import" and click on the "search form" link.
+![Workflow Import GIF](static/iwc-import.gif)
```

Comment:
Can you add a note under that animation saying: when searching against workflowhub.eu, please use organization:"IWC Workflows" as the search term?

Reply:
This needs to be fixed on the workflowhub side. iwc is the right organization name.

Reply:
Do you have a workflowhub account ? I do, but I can't log in, hanging in a loop. If you do, maybe you could submit the form https://workflowhub.eu/home/feedback ?

Reply:
Finally managed it in a private session.
<img width="1330" alt="Screenshot 2021-06-24 at 16 37 26" src="https://user-images.githubusercontent.com/6804901/123282219-961ffc80-d50a-11eb-8d35-9c0031b09812.png">

Reply:
Well, yes and no:

1. I don't quite understand what dockstore is doing either: when you look at the WFs returned by the organization:iwc query, they all list "iwc-workflows" as the organization (though undoubtedly we're iwc on the dockstore website). Not sure whether the query only works accidentally (organization:workflows seems to be working just as well).
2. Until this is unified upstream, we might still want to provide working instructions here (and simplify it later)?

Reply:
https://dockstore.org/organizations/iwc -> that is the organization name.
From the API:
```json
  {
    "aliases": {},
    "avatarUrl": "https://galaxyproject.org/images/galaxy-logos/galaxy_project_logo_square.png",
    "dbCreateDate": 1599229556066,
    "dbUpdateDate": 1624453421503,
    "description": "# IWC - Intergalactic Workflow Commission\n\n### Reproducibility is important. Our goals are to:\n - foster workflow use\n - incorporate versioning\n - capture more metadata: (names, versions, authors, use cases, etc.)\n - help scientists find workflows!",
    "displayName": "The Galaxy Intergalactic Workflow Comission",
    "email": "iwc@galaxyproject.org",
    "id": 33,
    "link": "https://github.com/galaxyproject/iwc",
    "location": null,
    "name": "iwc",
    "starredUsers": [
      {
        "id": 3
      },
      {
        "id": 47541
      },
      {
        "id": 53591
      }
    ],
    "status": "APPROVED",
    "topic": "This group exists to create and curate a registry of best practice community workflows for Galaxy.",
    "users": null
  },
```

Reply:
I think iwc-workflows is taken from the github namespace ... while I don't love it that makes sense.

Reply:
Reported that here: https://discuss.dockstore.org/t/organization-mismatch-if-github-app-installed-in-separate-namespace/4400.

I guess it doesn't hurt to add a note.

Reply:
Alright, with Denis' response I looked a bit closer, and searching for `iwc-workflows` should also return our workflows. So https://github.com/galaxyproject/galaxy/pull/12196 fixes that in Galaxy. So then I guess it's somewhat consistent. I updated the gif too.

Reply:
For information, when I follow exactly the gif instructions (Galaxy search form), the search works only with "iuc" and fails if I add "-workflows" or anything after "iuc"

Reply:
Thanks, yeah, that will be fixed in https://github.com/galaxyproject/galaxy/pull/12196

Reply:
@drosofff quoting the organization name is what's working right now, i.e. `organization:"iwc-workflows"` for dockstore and `organization:"IWC Workflows"` for workflowhub.

Reply:
I've deployed the update to usegalaxy.org, so organization:iwc-workflows works there.